validator/lib/validateTileset.js

```diff
@@ -168,3 +191,38 @@ function validateTileHierarchy(root, tilesetDirectory) {
             return message;
         });
 }
+
+function checkBoundingVolume(tileBV, parentBV, tileTransform, parentTransform, type) {
```

Comment:
Roughly yes. However lets avoid sending in `BoxinBox` and other strings. `checkBoundingVolume` should know the bounding volume types by using the `defined` checks like in the code above.

I think the code right now is constrained by how the error messages are created. `checkBoundingVolume` could return a generic message like `'box [' + box + '] is not within sphere [' + sphere + ']'`. Then since `checkBoundingVolume` is called by different areas (child vs parent and content vs tile), those can add onto the error message that is returned. Something like: `'content bounding volume is not within tile bounding volume:' + checkBoundingVolumeErrorMessage`.

With a few structural changes like this I think this area can become a lot cleaner.